src/App.js

```diff
@@ -32,8 +34,11 @@ function App() {
 				<Route path='/FirebaseTesting'>
 					<FirebaseTesting />
 				</Route>
+				{/*<Route path='/Home'>
+					<Home token={token} />
+				</Route>*/}
 			</Switch>
-
+                 <Home token={token}/>
```

Comment:
```suggestion
                 <Home token={token} setToken={setToken}/>
```
We need to also send setToken so the children components have access to it in Home.